scripts/ci/run_cli_tool.sh

```diff
@@ -157,12 +157,8 @@ RES=$?
 
 # Set file permissions to the host user
 if [[ "${HOST_OS}" == "Linux" ]]; then
-    FIX_DOCKER_ARGS=(--rm)
-    FIX_DOCKER_ARGS+=("${COMMON_DOCKER_ARGS[@]}")
-    FIX_COMMAND=(bash -c
-        "find ${DIRECTORIES_TO_FIX[@]@Q} -user root -print0 | xargs --null chown '${HOST_USER_ID}.${HOST_GROUP_ID}' --no-dereference")
-
-    docker run "${FIX_DOCKER_ARGS[@]}" "${AIRFLOW_CI_IMAGE}" "${FIX_COMMAND[@]}" >/dev/null 2>&1
+    find "${DIRECTORIES_TO_FIX[@]}" -user root -print0 2>/dev/null | \
```

Comment:
Docker is needed here to have access to the volume with credentials

Reply:
Correct. I simiplified it a bit and changed it to use the script we already had for that purpose. I also found out that we had another problem. "set +e" was missing so when the command failed, fixing ownership did not happen. I fixed that as well.

Pls. take a look @feluelle @mik-laj